src/main/java/net/mcreator/ui/dialogs/UpdateNotifyDialog.java

```diff
@@ -133,6 +135,46 @@ public static void showUpdateDialogIfUpdateExists(Window parent, boolean showNoU
 		}
 	}
 
+	public static void showPluginUpdateDialog(Window parent) {

```

Comment:
Move this to a new dialog class for plugin updates and I would suggest adding a plugin download URL so instead of opening the general page, a specific plugin page opens. Maybe make it possible to only open URLs starting with `https://mcreator.net/plugin/` as a security measure.